packages/amplify-category-storage/src/provider-utils/awscloudformation/index.ts

```diff
@@ -164,10 +164,8 @@ const isInHeadlessMode = (context: $TSContext) => {
 };
 
 const getHeadlessParams = (context: $TSContext) => {
-  const { inputParams } = context.exeInfo;
   try {
-    // If the input given is a string validate it using JSON parse
-    const { categories = {} } = typeof inputParams === 'string' ? JSONUtilities.parse(inputParams) : inputParams;
+    const { categories = {} } = context.exeInfo.inputParams;
```

Comment:
Could you:
- merge dev to pull latest eslint threshold
- address codeql warning
- if possible lower eslint warning threshold
?

Reply:
Should be resolved now.